tests/admin/dead_letter_queue/test_dlq.py

```diff
@@ -0,0 +1,5 @@
+from snuba.admin.dead_letter_queue import get_dlq_topics
+
+
+def test_dlq() -> None:
+    assert len(get_dlq_topics()) == 7
```

Comment:
I would probably test the structure of the response rather than this number. Otherwise this test will fail every time a new DLQ is added.